src/services/calculator-process.js

```diff
@@ -0,0 +1,71 @@
+let memory2 = [];
+function calculatorProcess(content,visor) {
```

Comment:
ok sounds good :) 

now you just need to refactor this code 😄 
think how you can improve this code without break the tests ;)

Reply:
it's done, this i like to refactor hehe kkkk
just because i remenbered one function i could use

Reply:
> it's done, this one i like to refactor hehe kkkk just because i remenbered one function i could use